### PR TITLE
docs: more details to url_for, relative_url & gravatar

### PR DESCRIPTION
**Pull request by curbengh**
## Check List

**Please read and check followings before submitting a PR.**

- [x] Others (Update, fix, translation, etc...)

Derived from recent commits of https://github.com/hexojs/hexo-util

Closes https://github.com/hexojs/site/issues/1085

Related https://github.com/hexojs/hexo-util/issues/87 https://github.com/hexojs/hexo-util/pull/88

### CODE REVIEW

**Comment by curbengh** [comment]
`url_for()` hasn't encode url yet.
Pending https://github.com/hexojs/hexo-util/pull/91 https://github.com/hexojs/hexo/pull/3710

File: source/docs/helpers.md [hunk 1]
```diff
@@ -25,15 +61,32 @@ Returns the relative URL from `from` to `to`.
...
 ### gravatar
 
-Inserts a Gravatar image.
+Returns the gravatar image url from an email.
+
 If you didn't specify the [options] parameter, the default options will apply. Otherwise, you can set it to a number which will then be passed on as the size parameter to Gravatar. Finally, if you set it to an object, it will be converted into a query string of parameters for Gravatar.
 
 ``` js
-<%- gravatar(email, [options]) %>
+<%- gravatar(email, [options] %>
```
Comment (SukkaW) [inline source/docs/helpers.md]:
```suggestion
<%- gravatar(email, [options]) %>
```

**Comment by SukkaW** [comment]
@curbengh rebase is required for this PR.

**Comment by NoahDragon** [comment]
@SukkaW thanks for pointing out the issue. I have resolved the conflicts and merged.

**Pull request merged**